source/main.c

```diff
@@ -8,6 +8,7 @@ extern void* nesinfo_data_start;
 extern void* my_malloc_current[2];
 extern void* my_zelda_malloc_align(unsigned long size, int align);
 extern void my_zelda_free(void *ptr);
+extern void* game_move_first;
```

Comment:
Please declare this as a function rather than pointer, e.g. `extern void  game_move_first();` (args are nice too but not necessary)